client/src/ui/PreviewCard.jsx

```diff
@@ -186,13 +186,13 @@ export default class PreviewCard extends Component {
 
         {!this.props.transformGauges && (
           <div className="row gaugeHolder">
-            <div className="col-md-4 col-sm-4 col-xs-4 remove-left-padding">
+            <div className="col-lg-4 col-md-4 col-sm-4 col-xs-4 remove-left-padding">
```

Comment:
```suggestion
            <div className="col-lg-4 col-md-4 col-sm-4 col-4 remove-left-padding">
```
there's no more `xs` breakpoint, so here and elsewhere `col-xs-{value}` should just be `col-{value}`